apps/krew-ui-angular/tailwind.config.js

```diff
@@ -1,18 +1,14 @@
 module.exports = {
-  prefix: '',
   mode: 'jit',
   purge: {
-    enabled: process.env.TAILWIND_MODE === 'build',
-    content: [
-      './apps/krew-ui-angular/**/*.{html,ts,css,scss,sass,less,styl}',
-      './libs/krew-angular/**/*.{html,ts,css,scss,sass,less,styl}',
-    ],
+    enabled: true,
+    content: ['apps/krew-ui-angular/**/*.{html,ts}', 'libs/krew-angular/**/*.{html,ts}'],
   },
   darkMode: 'media',
   theme: {
     extend: {
       colors: {
-        'brand-01': 'var(--brand-01)',
+        'brand-01': ({ opacityValue }) => `rgba(var(--brand-01), ${opacityValue})`,
```

Comment:
This is a very nice addition, but requires a change in the shared vars, but those weren't used by any other product, so _dips_!

Reply:
Created a new issue